homeassistant/components/calendar/todoist.py

```diff
@@ -365,7 +365,7 @@ def create_todoist_task(self, data):
 
         if not any(label in task[LABELS]
                    for label in self._label_whitelist) and (
-                   len(self._label_whitelist) > 0):
+                           len(self._label_whitelist) > 0):
```

Comment:
Don't check the length explicitly, just:
```py
if ... and self._label_whitelist:
```
An empty whitelist will evaluate to `False`.

Reply:
Even better actually would be to turn the expression around and check the whitelist first, then compare the whitelist labels with the task labels.